archetypes/Portfolio/History/index.tsx

```diff
@@ -162,82 +205,229 @@ const CommitRow: React.FC<
+                                {
+                                    type: ArbiscanEnum.txn,
+                                    target: transactionHashOut,
+                                    logo: ARBITRUM,
+                                    text: 'View Upkeep on Arbiscan',
+                                },
+                            ]}
+                        />
+                    </TableRowCell>
+                </TableRow>
+            );
+        } else if (type === COMMIT_TYPES_V2.LONG_BURN || type === COMMIT_TYPES_V2.SHORT_BURN) {
+            return (
+                <TableRow key={index} rowNumber={index}>
+                    <TableRowCell>
+                        <div>{timeString}</div>
+                        <div className="text-cool-gray-500">{dateString}</div>
+                    </TableRowCell>
+                    <TableRowCell>
+                        <div className="flex my-auto">
+                            <Logo
+                                size="lg"
+                                ticker={tokenSymbolToLogoTicker(tokenInSymbol)}
+                                className="inline my-auto mr-2"
+                            />
+                            <div>
+                                <div className="flex">
+                                    <div>
+                                        {tokenInSymbol.split('-')[0][0]}-
+                                        {marketSymbolToAssetName[tokenInSymbol.slice(3)]}
+                                    </div>
+                                    &nbsp;
+                                    <div className={type === COMMIT_TYPES_V2.LONG_BURN ? 'green' : 'red'}>
+                                        {type === COMMIT_TYPES_V2.LONG_BURN ? 'Long' : 'Short'}
+                                    </div>
+                                </div>
+                                {tokenInSymbol}
                             </div>
                         </div>
-                        {tokenSymbol}
-                    </div>
-                </div>
-            </TableRowCell>
-            {type === 'LongBurn' || type === 'ShortBurn' ? (
-                <>
+                    </TableRowCell>
                     <TableRowCell>
-                        <div>{tokenAmount.toFixed(2)} tokens</div>
+                        <div>{(tokenInAmount.toNumber() / 1e18).toFixed(2)} tokens</div>
                         <div className="text-cool-gray-500">
-                            at {toApproxCurrency(tokenPrice)} {collateralSymbol}/token
+                            at {toApproxCurrency(price.toNumber() / 1e18)} USD/token
+                        </div>
+                    </TableRowCell>
+                    <TableRowCell>
+                        {toApproxCurrency((price.toNumber() / 1e18) * (tokenInAmount.toNumber() / 1e18))}
+                    </TableRowCell>
+                    <TableRowCell>{toApproxCurrency(fee.toNumber() / 1e18)} USDC</TableRowCell>
```

Comment:
I dont think this will always be USDC so will have to use `quoteToken.symbol` here as well